src/containers/AccessPointDetails/components/Firmware/index.js

```diff
@@ -117,7 +117,12 @@ const Firmware = ({
       />
       <Form {...pageLayout} form={form} onValuesChange={handleOnFormChange}>
         <div className={styles.InlineEndDiv}>
-          <Button className={styles.saveButton} onClick={handleOnReboot} name="reboot">
+          <Button
+            className={styles.saveButton}
+            onClick={handleOnReboot}
+            name="reboot"
+            disabled={status.upgradeState !== 'undefined'}
```

Comment:
these are the possible states

[ undefined, download_initiated, downloading, download_failed, download_complete, apply_initiated, applying, apply_failed, apply_complete, reboot_initiated, rebooting, out_of_date, up_to_date, reboot_failed ]